relay-server/src/actors/processor.rs

```diff
@@ -520,8 +520,13 @@ impl FromMessage<RateLimitFlushBuckets> for EnvelopeProcessor {
 /// Service implementing the [`EnvelopeProcessor`] interface.
 ///
 /// This service handles messages in a worker pool with configurable concurrency.
+#[derive(Clone)]
 pub struct EnvelopeProcessorService {
     config: Arc<Config>,
+    inner: Arc<InnerEnvelopeProcessorService>,
+}
+
+struct InnerEnvelopeProcessorService {
```

Comment:
nit: Since this is a private type, I would be fine with a shorter name, e.g. `InnerProcessor`.